std/node/_internal/errors/errmap.ts

```diff
@@ -0,0 +1,30 @@
+      : unreachable(),
```

Comment:
This is pretty unreadable and I'm not sure how this is passing lint.

Please reformat this with 80 column maximum line width.

Reply:
Done!